preupg/settings.py

```diff
@@ -92,27 +89,21 @@
 # name of the file which contains a list of rules
 file_list_rules = "list_rules"
 
-# path to file with definitions of common scripts
-post_script = os.path.join(common_dir, "post_scripts.txt")
-
 # kickstart and postupgrade.d directories
-preupgrade_dirs = [dirty_conf_dir, clean_conf_dir,
-                   'hooks', 'kickstart', postupgrade_dir, 'common',
+preupgrade_dirs = [dirty_conf_dir, clean_conf_dir, 'hooks', 'data/templates',
+                   "data/" + postupgrade_dir, 'data/preassesment',
                    'preupgrade-scripts', 'noauto_postupgrade.d']
 
 
 PREUPG_README = 'README'
-readme_files = {'README': PREUPG_README,
-                'README.kickstart': os.path.join('kickstart', 'README'),
+readme_files = {'doc/README': PREUPG_README,
+                'doc/README.kickstart': os.path.join('kickstart', 'README'),
```

Comment:
This needs to be reverted. The two readmes are copied to /usr/share/preupgrade by setup.py directly, not to the doc subfolder.